sdk/src/system_transaction.rs

```diff
@@ -17,30 +34,78 @@ pub fn create_account(
     space: u64,
     program_id: &Pubkey,
 ) -> Transaction {
+    Transaction::new_signed_instructions(
+        &[from_keypair],
+        create_account_instructions(from_keypair, to, lamports, space, program_id),
+        recent_blockhash,
+    )
+}
+
+pub fn create_account_with_dummy_sig(
```

Comment:
[RFC] By no means, I'm not satified with these duplicated clumsy APIs.. At the moment, I've minimized the required change because this is used from a lot of places. Maybe prime time for the macro magic or more orthogonal API?

Reply:
Now I think this is fine as it is now.